scripts/cardano-node.sh

```diff
@@ -9,7 +9,7 @@
 
 set -euo pipefail
 
-node_zipped="cardano-node-1.34.1-linux.tar.gz"
+node_zipped="cardano-node-1.35.2-linux.tar.gz"
```

Comment:
1.35.3 was out now right?

Reply:
No, not yet. Also, cardano-wallet has only updated until 1.35.2